works-councils/works-councils.md

```diff
@@ -27,3 +27,9 @@ Please check the [Events](/events) section for upcoming times and registration f
 Below we collect articles on Works Council activities in Berlin tech companies. In an interview series, we present perspectives and experiences from Works Councillors. If you are a Works Councillor yourself, or know a fellow worker who is, and would be interested taking part, we'd love to hear from you: [content@techworkersberlin.com](mailto:content@techworkersberlin.com).
 
 {% include news.html tags=page.tags %}
+
+<span id="resources" />
+
+## Resources
+
+* [Codetermination in the Workplace in Germany](https://www.dgb.de/betriebsratswahl/++co++97a86fd0-70bb-11ed-865c-001a4a160123) (available in 11 languages)
```

Comment:
Completely makes sense to add here (would be https://techworkersberlin.com/works-councils in the end). I was additionally thinking about
*  https://techworkersberlin.com/ru 
* https://techworkersberlin.com/pl

Since those are two links with very limited German labour info/content (both on this website and in general) and there to take advantage of improved SEO by including local language excerpts with an exact link to the PDF (with smaller risk of link rot)

Reply:
@shushugah oh that makes so much sense! I feel silly not thinking of the other locales. So [techworkersberlin.com/ru/works-councils](https://techworkersberlin.com/ru/works-councils) for example would have this link? or the base path?

Reply:
@shushugah ah ok, i think I understand now - so you want to add this to the `_i18n/{lang}.yml` files then